src/Integration/ErrorListenerIntegration.php

```diff
@@ -61,7 +69,7 @@ public function setupOnce(): void
                 return;
             }
 
-            Hub::getCurrent()->captureException($exception);
+            $this->hub->captureException($exception);
```

Comment:
This changes the behavior: the hub is now bound at construction and, if the main is changed afterwards, this integration will not acknowledge that. Is this intended?

Reply:
>Is this intended?

No, this is an issue I didn't account for when coding. I will revert the change